keanu-project/src/main/java/io/improbable/keanu/distributions/discrete/Poisson.java

```diff
@@ -78,27 +77,10 @@ private static int sample(double mu, KeanuRandom random) {
 
     @Override
     public DoubleTensor logProb(IntegerTensor k) {
```

Comment:
no wait... this is the one I wanna put on the wall! 20loc --> 3loc 😮

Reply:
Yeah it's a huge trim. Not sure why we had to do it the flatten way before...